scenarios/ocd/CreateWordPressDeployment/README.md

```diff
@@ -0,0 +1,466 @@
+
+## Define Environment Variables
+
+The First step in this tutorial is to define environment variables.
+
+```bash
+export SSL_EMAIL_ADDRESS="$(az account show --query user.name --output tsv)"
+export NETWORK_PREFIX="$(($RANDOM % 253 + 1))"
+export RANDOM_ID="$(openssl rand -hex 3)"
+export MY_RESOURCE_GROUP_NAME="myResourceGroup$RANDOM_ID"
+export REGION="eastus"
+export MY_AKS_CLUSTER_NAME="myAKSCluster$RANDOM_ID"
+export MY_PUBLIC_IP_NAME="myPublicIP$RANDOM_ID"
+export MY_DNS_LABEL="mydnslabel$RANDOM_ID"
+export MY_VNET_NAME="myVNet$RANDOM_ID"
+export MY_VNET_PREFIX="10.$NETWORK_PREFIX.0.0/16"
+export MY_SN_NAME="mySN$RANDOM_ID"
+export MY_SN_PREFIX="10.$NETWORK_PREFIX.0.0/22"
+export MY_MYSQL_DB_NAME="mydb$RANDOM_ID"
+export MY_MYSQL_ADMIN_USERNAME="dbadmin$RANDOM_ID"
+export MY_MYSQL_ADMIN_PW="$(openssl rand -base64 32)"
+export MY_MYSQL_SN_NAME="myMySQLSN$RANDOM_ID"
+export MY_MYSQL_HOSTNAME="$MY_MYSQL_DB_NAME.mysql.database.azure.com"
+export MY_WP_ADMIN_PW="$(openssl rand -base64 32)"
+export MY_WP_ADMIN_USER="wpcliadmin"
+export FQDN="${MY_DNS_LABEL}.${REGION}.cloudapp.azure.com"
+```
+
+## Create a resource group
+
+A resource group is a container for related resources. All resources must be placed in a resource group. We will create one for this tutorial. The following command creates a resource group with the previously defined $MY_RESOURCE_GROUP_NAME and $REGION parameters.
+
+```bash
+az group create \
+    --name $MY_RESOURCE_GROUP_NAME \
+    --location $REGION
+```
+
+Results:
+
+<!-- expected_similarity=0.3 -->
+```json
+{
+  "id": "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/myResourceGroup210",
+  "location": "eastus",
+  "managedBy": null,
+  "name": "testResourceGroup",
+  "properties": {
+    "provisioningState": "Succeeded"
+  },
+  "tags": null,
+  "type": "Microsoft.Resources/resourceGroups"
+}
+```
+
+## Create a virtual network and subnet
+
+A virtual network is the fundamental building block for private networks in Azure. Azure Virtual Network enables Azure resources like VMs to securely communicate with each other and the internet.
+
+```bash
+az network vnet create \
+    --resource-group $MY_RESOURCE_GROUP_NAME \
+    --location $REGION \
+    --name $MY_VNET_NAME \
+    --address-prefix $MY_VNET_PREFIX \
+    --subnet-name $MY_SN_NAME \
+    --subnet-prefixes $MY_SN_PREFIX
+```
+
+Results:
+
+<!-- expected_similarity=0.3 -->
+```json
+{
+  "newVNet": {
+    "addressSpace": {
+      "addressPrefixes": [
+        "10.210.0.0/16"
+      ]
+    },
+    "enableDdosProtection": false,
+    "id": "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/myResourceGroup210/providers/Microsoft.Network/virtualNetworks/myVNet210",
```

Comment:
Can we make the whole id in the result blocks generic?  myVNet210 -> myVNetxxx, mSN210 -> mySNxxx, etc.